src/rpc/misc.cpp

```diff
@@ -449,6 +451,36 @@ static UniValue getmemoryinfo(const JSONRPCRequest& request)
     }
 }
 
+static UniValue getgeneralinfo(const JSONRPCRequest& request)
+{
+    RPCHelpMan{"getgeneralnfo",
+                "\nReturns data about the bitcoin daemon.\n",
+                {},
+                RPCResult{
+                "  {\n"
+                "    \"clientversion\": \"version string\",     (string) Client version\n"
+                "    \"useragent\":\"client name\",             (string) Client name\n"
+                "    \"datadir\":\"datadir path\",              (string) Data directory path\n"
+                "    \"blocksdir\":\"blocksdir path\",          (string) Blocks directory path\n"
+                "    \"startuptime\":\"string\",                (string) Startup time\n"
```

Comment:
`uptime` in `server.cpp` returns type `int64_t`. I think for consistency it would be better for `startuptime` to return the same thing. The consumer of the timestamp can choose how to format it.

Reply:
Well, the GUI shows a properly formatted / localized time. Returning a numeric value would maybe collide with user expectations? Or we could simply return two values (raw number and localized one). In any case, this should be not a big problem. Let's wait for more input, shall we? :)